Index: .idea/misc.xml
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
deleted file mode 100644
--- a/.idea/misc.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/OOP.iml
===================================================================
diff --git a/.idea/OOP.iml b/.idea/OOP.iml
deleted file mode 100644
--- a/.idea/OOP.iml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: .idea/modules.xml
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
deleted file mode 100644
--- a/.idea/modules.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/OOP.iml" filepath="$PROJECT_DIR$/.idea/OOP.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: .idea/vcs.xml
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
deleted file mode 100644
--- a/.idea/vcs.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: Java/First seminar/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java/First seminar/.idea/misc.xml b/Java/First seminar/.idea/misc.xml
deleted file mode 100644
--- a/Java/First seminar/.idea/misc.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
Index: Java/First seminar/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java/First seminar/.idea/vcs.xml b/Java/First seminar/.idea/vcs.xml
deleted file mode 100644
--- a/Java/First seminar/.idea/vcs.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$/../.." vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
Index: Java/First seminar/.idea/workspace.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java/First seminar/.idea/workspace.xml b/Java/First seminar/.idea/workspace.xml
deleted file mode 100644
--- a/Java/First seminar/.idea/workspace.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,62 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ChangeListManager">
-    <list default="true" id="695a45a0-77e1-401b-b742-853d6cba668c" name="Changes" comment="">
-      <change beforePath="$PROJECT_DIR$/../../C - projects/this" beforeDir="false" />
-      <change beforePath="$PROJECT_DIR$/../Readme.md" beforeDir="false" />
-    </list>
-    <option name="SHOW_DIALOG" value="false" />
-    <option name="HIGHLIGHT_CONFLICTS" value="true" />
-    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
-    <option name="LAST_RESOLUTION" value="IGNORE" />
-  </component>
-  <component name="Git.Settings">
-    <option name="RECENT_GIT_ROOT_PATH" value="$PROJECT_DIR$/../.." />
-  </component>
-  <component name="ProjectColorInfo"><![CDATA[{
-  "associatedIndex": 5
-}]]></component>
-  <component name="ProjectId" id="2cMDf2z4PZLyVKVGFlwbuRjz19p" />
-  <component name="ProjectViewState">
-    <option name="hideEmptyMiddlePackages" value="true" />
-    <option name="showLibraryContents" value="true" />
-  </component>
-  <component name="PropertiesComponent"><![CDATA[{
-  "keyToString": {
-    "RunOnceActivity.OpenProjectViewOnStart": "true",
-    "RunOnceActivity.ShowReadmeOnStart": "true",
-    "git-widget-placeholder": "main",
-    "kotlin-language-version-configured": "true",
-    "nodejs_package_manager_path": "npm",
-    "onboarding.tips.debug.path": "/Users/benjaminptacek/Library/CloudStorage/OneDrive-SlovenskátechnickáuniverzitavBratislave/Škola/2. Ročník/2. semester/OOP/Java/First seminar/src/Main.java",
-    "vue.rearranger.settings.migration": "true"
-  }
-}]]></component>
-  <component name="RunManager">
-    <configuration default="true" type="GroovyScriptRunConfiguration" factoryName="Groovy">
-      <setting name="path" value="" />
-      <setting name="vmparams" value="" />
-      <setting name="params" value="" />
-      <setting name="workDir" value="file://$PROJECT_DIR$" />
-      <setting name="debug" value="false" />
-      <method v="2">
-        <option name="Make" enabled="true" />
-      </method>
-    </configuration>
-  </component>
-  <component name="SpellCheckerSettings" RuntimeDictionaries="0" Folders="0" CustomDictionaries="0" DefaultDictionary="application-level" UseSingleDictionary="true" transferred="true" />
-  <component name="TaskManager">
-    <task active="true" id="Default" summary="Default task">
-      <changelist id="695a45a0-77e1-401b-b742-853d6cba668c" name="Changes" comment="" />
-      <created>1707912655446</created>
-      <option name="number" value="Default" />
-      <option name="presentableId" value="Default" />
-      <updated>1707912655446</updated>
-      <workItem from="1707912656486" duration="58000" />
-    </task>
-    <servers />
-  </component>
-  <component name="TypeScriptGeneratedFilesManager">
-    <option name="version" value="3" />
-  </component>
-</project>
\ No newline at end of file
Index: Java/First seminar/First seminar.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java/First seminar/First seminar.iml b/Java/First seminar/First seminar.iml
deleted file mode 100644
--- a/Java/First seminar/First seminar.iml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: Java/First seminar/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Java/First seminar/.idea/modules.xml b/Java/First seminar/.idea/modules.xml
deleted file mode 100644
--- a/Java/First seminar/.idea/modules.xml	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
+++ /dev/null	(revision 5f93a3e49fc687a552167cf25cb719d2f01ccec4)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/First seminar.iml" filepath="$PROJECT_DIR$/First seminar.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
diff --git a/Java/First seminar/First seminar/.gitignore b/Java/First seminar/.gitignore
rename from Java/First seminar/First seminar/.gitignore
rename to Java/First seminar/.gitignore
diff --git a/Java/First seminar/First seminar/src/Main.java b/Java/First seminar/src/Main.java
rename from Java/First seminar/First seminar/src/Main.java
rename to Java/First seminar/src/Main.java
